modules/optimization/examples/materialTransient/optimize_grad.i

```diff
@@ -0,0 +1,88 @@
+[Optimization]
+[]
+
+[OptimizationReporter]
+  type = OptimizationReporter
+  parameter_names = 'D'
+  num_values = '4'
+  initial_condition = '1 1 1 1'
```

Comment:
Change lines 8-10 to:
  initial_condition = '0.01 0.01 0.01 0.01'
  upper_bounds = '1e2'
  lower_bounds = '1e-3'

Reply:
Updated!